sqlx-postgres/src/copy.rs

```diff
@@ -188,13 +188,18 @@ impl<C: DerefMut<Target = PgConnection>> PgCopyIn<C> {
     ///
     /// If you're copying data from an `AsyncRead`, maybe consider [Self::read_from] instead.
     pub async fn send(&mut self, data: impl Deref<Target = [u8]>) -> Result<&mut Self> {
```

Comment:
It's probably worth documenting that the data is sent in chunks and may be partially sent if this call is cancelled.